maestro-test/src/main/kotlin/maestro/test/drivers/FakeDriver.kt

```diff
@@ -162,6 +162,12 @@ class FakeDriver : Driver {
         events += Event.BackPress
     }
 
+    override fun hideKeyboard() {
```

Comment:
Could you please add a new test case in `IntegrationTest`?

Same pattern as with other tests there:
- Create a `yaml` file where you invoke `hide keyboard`
- Load this file in a test
- Run it on a `FakeDriver`
- Assert that `HideKeyboard` event was received

Reply:
updated on this commit [fbc5007](https://github.com/mobile-dev-inc/maestro/pull/165/commits/fbc50075fbf271eb763cb3daf07cb44d47a574f0)